go.mod

```diff
@@ -12,15 +12,19 @@ require (
 	github.com/sirupsen/logrus v1.8.1
 	github.com/spf13/cobra v1.2.1
 	github.com/spf13/pflag v1.0.5
+	github.com/stretchr/testify v1.7.0
```

Comment:
If the maintainers are against a new dependency, I will update the new tests to manually assert conditions.  I am a big fan of testify, though.  Would recommend!  😁

Reply:
I would shy away from this dependency, not per se because it's a new dependency, but because it goes against established Go guidelines to use an assertion library, and I think it's valuable to try to follow Go guidelines, since we're kind of a cousin project. 

https://github.com/golang/go/wiki/TestComments#assert-libraries

But I'm interested to hear if @nevkontakte feels differently.

Reply:
Fair enough, I won't make waves.